scripts/install-verus.sh

```
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+set -eu
+
+SCRIPT_DIR=$(CDPATH= cd -- "$(dirname -- "$0")" && pwd)
+REPO_ROOT=$(CDPATH= cd -- "${SCRIPT_DIR}/.." && pwd)
```

Shellcheck (0.11.0)

[warning] REPO_ROOT appears unused. Verify use (or export if used externally).

(SC2034)